css/mobile.css

```diff
@@ -0,0 +1,133 @@
+header {
+  position: sticky;
+  top: 0;
+  display: flex;
+  width: 100%;
+  flex-flow: row nowrap;
+  justify-content: center;
+  align-items: center;
+  min-height: 10vh;
+}
+
+.brand {
+  display: inline-block;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  margin-left: 1rem;
+  color: var(--site-color);
+  flex: 200px 1;
+}
+
+.nav-button {
+  display: none;
+  background: url(../img/menu.svg);
+  background-repeat: no-repeat;
+  background-size: 50% 50%;
+  background-position: 50%;
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+  margin-right: 1rem;
+  z-index: 4;
+}
+
+.nav-button:hover {
+  box-shadow: 0 0 1px 1px lightgray;
+  border-radius: 2px;
+}
+
+#hidden-toggler {
+  display: none;
+}
+
+#hidden-toggler:checked ~ .nav-button {
+  background: url(../img/close.svg);
+  background-repeat: no-repeat;
+  background-size: 50% 50%;
+  background-position: 50%;
+}
+
+.nav {
+  position: static;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  align-items: center;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  list-style: none;
+  flex: 200px 9;
+}
+
+.nav-link {
+  text-decoration: none;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  color: black;
+  display: inline-block;
+  padding: 0.5rem;
+}
+
+main {
+  margin: 1.5rem;
+}
+
+.connect {
+  display: flex;
```

Comment:
We can try to put some repetitive codes into a class and add the class in the HTML tag.
Example
.d-flex{display:flex}

Then add a class of d-flex to any section that needs to be flexed.